<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show Eraser Config</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="cursorFileLabel">
       <property name="text">
        <string>Custom Cursor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cursorFileBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="folder"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearCursorFileBtn">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="delete"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="boolEnableCursorAdjust">
     <property name="title">
      <string>Enable Cursor adjust based on zoom level</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Less Than:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="intLessThan">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="lessThanAdjustCmb"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Default Size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="defaultSizeCmb"/>
          </item>
          <item>
           <widget class="QComboBox" name="defaultSizeAdjustCmb"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>More Than:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="intMoreThan">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="moreThanAdjustCmb"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
